test/e2e/antreapolicy_test.go

```diff
@@ -1220,3 +1222,118 @@ func TestAntreaPolicyStatus(t *testing.T) {
 	})
 	assert.NoError(t, err, "Antrea ClusterNetworkPolicy failed to reach expected status")
 }
+
+// TestANPNetworkPolicyStatsWithDropAction tests traffic from busybox to nginx on port 80, after applying specific ANP
+// with drop aciton. Also it tests that antreanetworkpolicystats can correctly collect dropped packets stats from ANP if
```

Comment:
I didn't see it tested traffic, and least it didn't verify it. And perhaps it's redundant to verify the correctness of drop action here as it has been covered by other cases. We could just say the test focus on anp stats with drop action, or anp stats with both allow and drop actions.

Reply:
The traffic of AntreaNetworkPolicy(drop action)  is covered by https://github.com/vmware-tanzu/antrea/blob/c509f5cc670ac6a08b355f59586ca204dfdfb5fb/test/e2e/antreapolicy_test.go#L945 in  `testANPBasic` so yes, it is redundant. I will modify the test description later